src/pages/404.tsx

```diff
@@ -7,12 +7,11 @@ import { classes } from "./_error.st.css";
 
 const Error404: NextPage = () => {
 	const { siteTitle, siteSubtitle } = useContext(ReactLocaleContext);
+	const pageTitle = `${siteTitle} - ${siteSubtitle} - 404`;
 	return (
 		<Layout>
 			<Head>
-				<title>
-					{siteTitle} - {siteSubtitle} - 404
-				</title>
+				<title>{pageTitle}</title>
```

Comment:
How about a title component, to replace the repetition of `useContext ` and `const `this and that?

Reply:
@imdfl - The title fix is to avoid warnings on build. What you describe is an optimization for a later PR. Please approve this PR so we can get rid of the bug in resources page, which is priority.